### PR TITLE
[CI] - Add umami image cron build

### PR DESCRIPTION
**Pull request by yqlbu**
### Summary

Introduce umami image cron build

### Description

Umami is a simple, fast, website analytics alternative to Google Analytics.

### Details

- [x] Added CI cron-build for umami
- [x] Added usage cases and installation guide in README.md

### References

- https://github.com/mikecao/umami

### Checks

- [x] Already tested changes in `staging` environment

### Notes

- _Some awesome notes_

### CODE REVIEW

**Review by kunish — CHANGES_REQUESTED** [review]
Maybe it's a better idea that we use the upstream source code directly with cronjob.

Something like this

```yml
jobs:
  build:
     steps:
       - run: |
           git clone https://github.com/mikecao/umami.git
           docker build . -t ghcr.io/techprober/umami
           docker push $_
```

**Review by kunish — APPROVED** [review]
LGTM

**Comment by yqlbu** [comment]
We should add more details about usage case in `README.md`

File: containers/umami/docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+version: "3"
+services:
+  umami:
+    image: ghcr.io/techprober/umami:latest
```
Comment (yqlbu) [inline containers/umami/docker-compose.yaml:5]: I modified the image name pointing to our `GHCR` registry

**Comment by yqlbu** [comment]
Already tested in `staging` environment, looks like it is quite stable. Good to proceed to `production`.

**Review by kunish — APPROVED** [review]
LGTM

**Pull request merged**